module/main.c

```diff
@@ -131,13 +127,13 @@ int16_t output, output_new;
 #define FIRSTRUN_KEY 0x22
 
 typedef const struct {
-    tele_script_t script[10];
-    tele_pattern_t patterns[4];
+    scene_script_t script[10];
```

Comment:
curious about the name change - `scene_script_t` is still a single script, right? 'scene' here seems to be more appropriate for the variable name (an array of scripts for a scene) than the type itself. 

also thinking might be good to rename `script` to `scripts` to keep it consistent with `patterns`. this is a minor thing though, as is the above.